stable/drone/README.md

```diff
@@ -1,27 +1,48 @@
 # Drone.io
 
-[Drone](http://readme.drone.io/) is a Continuous Integration platform built on container technology.
+[Drone](http://readme.drone.io/) v1 is a Continuous Integration platform built on container technology with native Kubernetes support.
+
+> It is not recommended to upgrade from earlier (0.8.x) versions of Drone due to the large amount of breaking changes both in the product and in the helm charts.
 
 ## TL;DR;
 
 ```console
-$ helm install stable/drone
+helm install stable/drone
 ```
 
 ## Installing the Chart
 
 To install the chart with the release name `my-release`:
 
 ```console
-$ helm install --name my-release stable/drone
+helm install --name my-release stable/drone
+```
+
+> note: The chart will not install the drone server until you have configured a source control option. If this is the case it will print out notes on how to configure it in place using `helm upgrade`.
+
+An example (secrets redacted) working install of the chart using github as the source control provider:
+
+```console
+$ helm install --name drone --namespace drone stable/drone
```

Comment:
@paulczar you removed the $ from the examples above, might want to remove from those here to be consistent.

Reply:
good catch!  fixed.